appveyor.yml

```diff
@@ -0,0 +1,15 @@
+version: '{build}'
+build_script:
+- cmd: >-
+    set RUBY_VERSION=200
+
+    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+
+    gem install bundler
+
+    bundle install
+
+    build.netstd.cmd
+test: off
+on_finish:
+    - ps: $root = Resolve-Path .\artifacts; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }

```

Comment:
What does this line do?

If I "Export YAML" from the AppVeyor UI, I get:

```yaml
version: '{build}'
environment:
  VERSION_SUFFIX: -netstd
test: off
artifacts:
- path: artifacts/output/*
```

Is the `artifacts` setting not sufficient?